frontend/src/export/text.js

```diff
@@ -0,0 +1,24 @@
+import {ZONE_MAIN, ZONE_SIDEBOARD} from "../zones";
+
+export default {
+  name: "text",
```

Comment:
I think it looks nicer when buttons use capital styling --> `Text`

Reply:
Sure, can change this

Reply:
Done